docs/user-guide/timeline.md

```diff
@@ -1,114 +1,90 @@
 
 ### Animation Settings
 
-The animation behavior can be customized through the **Settings**. If you click on the button <img src="../img/button/timeline-playback-settings-button.jpg" class="ms-docbutton"/> a panel with two different sections appears.
+The animation behavior can be customized through the **Settings** button <img src="../img/button/timeline-playback-settings-button.jpg" class="ms-docbutton"/> allowing the user to setting the *Timeline* and the *Playback*. 
 
 <img src="../img/timeline/timeline-animation-settings.jpg" class="ms-docimage"  style="max-width:500px;" />
 
-***Timeline Settings***
+By default, the **Snap to guide layer** is enabled allows to force the time cursor to snap to the selected layer's data. The user can disable *Snap to guide level* to select the time cursor he prefers through the section **Animation passage**, for example, the process could be similar to the following: 
 
-* **Snap to guide layer**: this option is active by default. It forces the time cursor to snap to the selected layer's data. Disabling this option the **Animation Step** selection in the ***Playback Settings*** section becomes available.
+<img src="../img/timeline/animation-passage.jpg" class="ms-docimage"/>
 
-***Playback Settings***
+The user can set the second between one frame and another through the **Frame Duration** and enable the **Follow the animation** to visualize the animation also on the histogram.
 
-* **Frame Duration**: it indicates how fast the time cursor switch to the next step in the steps sequence
+<img src="../img/timeline/frame-duration.jpg" class="ms-docimage"/>
 
-* **Animation Step**: it defines the step width and so all the steps positions along the time axis. By default this option is disabled and the step sequence is defined by the ***guide layer***'s data. If activated, a fixed time unit (in terms of seconds, minutes, hours, days, months or years) can be defined as the step width and the whole steps sequence will be reconfigured.
+Enabling the  **Animation Range** the user can bound the animation execution to a fixed time interval, the *green range*. The *green range* can be defined both dragging the *play/stop cursors* directly on the histogram or filling the *date/time control cells* of the extra panel displayed, as follows:
 
-* **Animation Range**: if activated, it basically bounds the animation execution to a fixed time interval, the *green range*. The *green range* can be defined both dragging the *play/stop cursors* directly on the histogram or filling the *date/time control cells* of the extra panel displayed.
-    <br>Other settings:
+<img src="../img/timeline/timeline-animation-green-range.gif" class="ms-docimage"/>
 
-    * **Zoom to the current playback range** <img src="../img/button/timeline-zoom-playback-range.jpg" class="ms-docbutton"/> : zoom the histogram until it fits the animation's *green range* time extension.
+In order to set this filter the user can:
```

Comment:
In order to properly set the Animation Ranger, some controls are available to help the user.

The following option is missing:

Fit to selected layer's range*  : extend the animation's green range until it fits the guide layer*** time extension